spec/restaurant_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'restaurant'
+
+describe Restaurant do
+
+let(:menu_klass) {double :menu_klass}
+let(:menu) {double :menu}
+let(:text_klass) {double :text_klass}
+
+before do
+  allow(menu_klass).to receive(:new)
+  allow(menu_klass).to receive(:list).and_return(:item_1 => 5, :item_2 => 3)
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)